### PR TITLE
Update New-CsTeamsMeetingPolicy.md

### PR DESCRIPTION
**Pull request by bwilsonms**
Adding parameter for CART captions

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -344,6 +344,24 @@ Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
+### -AllowCartCaptionsScheduling
+Determines whether a user can add a URL for captions from a Communicatons Access Real-Time Translation (CART) captioner for providing real time captions in meetings.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:348]:
```suggestion
Determines whether a user can add a URL for captions from a Communications Access Real-Time Translation (CART) captioner for providing real-time captions in meetings.
```

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 2]
```diff
@@ -344,6 +344,24 @@ Default value: None
...
+Possible values are:
+- **EnabledUserOverride**, CART captions is available by default but a user can disable.
+- **DisabledUserOverride**, if you would like users to be able to use CART captions in meetings but by default it is disabled. 
+- **Disabled**, if you'd like to not allow CART captions in meeting.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:352]:
```suggestion
- **EnabledUserOverride**: CART captions are available by default but you can disable them.
- **DisabledUserOverride**: If you would like users to be able to use CART captions in meetings but they are disabled by default. 
- **Disabled**: If you do not want to allow CART captions in meetings.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @bwilsonms, kindly commit the suggested changes. Many thanks.

**Review by dariomws — APPROVED** [review]
I'm ok after @JohanFreelancer9 suggestions.

**Comment by dariomws** [comment]
Could we also apply @JohanFreelancer9 suggestions to the Set- cmdlet?

**Comment by yogkumgit** [comment]
@bwilsonms

**Comment by bwilsonms** [comment]
Done thanks all I was off yesterday.

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9 @get-itips

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by bwilsonms** [comment]
Thanks all!

**Pull request merged**